index.markdown

```diff
@@ -46,6 +46,11 @@ secondary:
   content: "Check out the [forums](https://discuss.babelfishpgcommunity.dev) to stay informed."
 
 sidebar:
+  - title: Releases
+    description: List of Babelfish releases.
+    links:
+      - title: Release 1.0.0
+        url: "/versions/babelfish-1-0-0.html"
   - title: Have a question?
     description: Do you have a question regarding Babelfish? Maybe you aren’t the first to ask it. Checkout our frequently asked questions (FAQ) to see if your question is already answered. If not, don’t hesitate to ask in the community forums.
```

Comment:
How about:

description: Do you have a question about Babelfish? You may not be the first person to ask it. Visit our Frequently Asked Questions (FAQ) to see if your question has been answered. If not, don’t hesitate to ask, and we'll be glad to answer!